catalog/service/rest/src/main/java/org/projectnessie/catalog/service/rest/IcebergErrorMapper.java

```diff
@@ -89,6 +93,33 @@ public Response toResponse(Throwable ex, IcebergEntityKind kind) {
     return Response.status(code == null ? 500 : code).entity(body).build();
   }
 
+  private static String message(ObjectIOException e) {
+    return String.format(
+        "Unable to %s %s due to: %s",
+        (e.ioMode().name().toLowerCase(Locale.ROOT)),
```

Comment:
Maybe add the lower-case representation via a function on the enum to avoid the `.toLowerCase`

Reply:
not used anymore